vivisect/analysis/generic/emucode.py

```diff
@@ -71,6 +73,15 @@ def prehook(self, emu, op, eip):
             emu.stopEmu()
             raise v_exc.BadOutInstruction(op.va)
 
+        if op.mnem == 'int':
```

Comment:
This is kinda the cleanest and earliest place we can put this in order for codeflow to recognize that `int 0x80` on linux with a stack param of 1 is a no return case.

Reply:
So would I, but it's either it lives here, or we break the emucode pass into arch-specific ones and duplicate code over there just for this. I chose this here because I need the emulator to not run into lalaland after we encounter instructions like this and I don't have to have wholesale copy this file just to add one condition.

I can section these out to a dictionary lookup like you suggested for the archmod, but these want to live here to make other features work properly.

Reply:
please see my other comment on IF_INTERRUPT.
please don't do string comparisons here.  compare iflags again IF_INTERRUPT (or pick another name/paradigm).